dlt/cli/helpers/dependencies.py

```diff
@@ -0,0 +1,71 @@
+import importlib
+import sys
+from subprocess import check_call, CalledProcessError
+from dlt.common.exceptions import MissingDependencyException
+
+
+def install_missing_package(package):
+    try:
+        importlib.import_module(package)
+    except ImportError:
+        try:
+            check_call(["pip", "install", package])
+        except CalledProcessError as e:
+            print(f"Could not install {package}")
+            sys.exit(e.returncode)
+    finally:
+        globals()[package] = importlib.import_module(package)
+
+
+def query_yes_no(question, default="yes"):
```

Comment:
what's wrong with current `echo.py` and `clikck` library we use. this should IMO work